api/formatters/geojson.py

```diff
@@ -8,13 +8,16 @@ def _make_properties(ts):
     ts_metadata = {key.name: value for key, value in ts.ts_mdata.ListFields() if value}
 
     ts_metadata["platform_vocabulary"] = (
-        "https://oscar.wmo.int/surface/#/search/station/stationReportDetails/" + ts.ts_mdata.platform
+        "https://oscar.wmo.int/surface/rest/api/search/station?wigosId=" + ts.ts_mdata.platform
         if not ts.ts_mdata.platform_vocabulary
         else ts.ts_mdata.platform_vocabulary
     )
-    # This should also be compatible with future when name is added to datastore
-    if "name" not in ts_metadata:
-        ts_metadata["name"] = ts.ts_mdata.platform  # TODO: grab proper name when implemented in proto
+
+    if "platform_name" in ts_metadata:
+        ts_metadata["name"] = ts_metadata["platform_name"]
+        del ts_metadata["platform_name"]
+    else:
+        ts_metadata["name"] = f'platform-{ts_metadata["platform"]}'
```

Comment:
Keep this `platform_name`, as we are listing time series here. Using `name` would be confusing.

Reply:
Changed back to `platform_name` in `/items`